test/Microsoft.AspNet.Mvc.Core.Test/ApplicationModel/ActionModelTest.cs

```diff
@@ -78,6 +79,13 @@ public void CopyConstructor_CopiesAllProperties()
                     // Ensure non-default value
                     Assert.NotEmpty((IEnumerable<object>)value1);
                 }
+                else if (typeof(IDictionary<object, object>).IsAssignableFrom(property.PropertyType))
```

Comment:
The point of doing reflection is that the test fails if you add something and don't update the test. You can't miss